src/App.js

```diff
@@ -13,11 +13,20 @@ function App() {
   return(
     <div>
       <input
-        type="search"
+        type='search'
         onInput={handleInput}
+        className='search-bar'
+        placeholder='Search wiki'
       />
-      {articles.map((article,i) => (
-        <li key={i}>{article}</li>
+      {articles.map((article) => (
+        Array.isArray(article)
```

Comment:
If you use the default value as an array in the `useFetch` callback you don't need this check for Array.isArray

Reply:
The first value of the response is the search term which is the string. SO i think this check is necessary

Reply:
No I meant if you did
```
  let articles = useFetch(
    `https://en.wikipedia.org/w/api.php?action=opensearch&origin=*&search=${searchTerm}`,
     []
  )
```

You don't need the Array.isArray check